### PR TITLE
chage the name of file from VikRam.json to vikram-8290.json

### PR DESCRIPTION
**Pull request by vikram-8290**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6678"></a>

### CODE REVIEW

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
You have added two profiles, please remove the one that does not match your GitHub username.

File: data/vikram-8290
Comment (HassanTanveer) [inline data/vikram-8290:1]: Please rename this file to `vikram-8290.json`

File: data/VikRam.json
Comment (HassanTanveer) [inline data/VikRam.json:1]: You may delete this file

**Comment by SaraJaoude** [comment]
@HassanTanveer please avoid duplicating comments.  This causes confusion to the author as well as everyone else involved, plus it generates unnecessary notifications for the maintainers.

File: data/VikRam.json
Comment (vikram-8290) [reply data/VikRam.json:1]: how can i delete this file.please help me this

**Comment by vikram-8290** [comment]
> You have added two profiles, please remove the one that does not match your GitHub username.

i am new github please tell me how can i remove the other one

**Pull request closed without merging**